src/main/java/com/faforever/client/discord/DiscordRichPresenceService.java

```diff
@@ -93,6 +93,8 @@ public void updatePlayedGameTo(Game game, int currentPlayerId, String currentPla
 
       if (game.getStartTime() != null) {
         discordRichPresence.setStartTimestamps(game.getStartTime().toEpochMilli());
+      } else if (game.getStatus() == GameStatus.PLAYING) {
+        discordRichPresence.setStartTimestamps(Instant.now().toEpochMilli());
```

Comment:
This is confusing me.
a) When would this case happen?
b) As you can not join a running game, the start timestamp is still the start time of the game, isn't it?

Reply:
Well the start time is sometimes set by the server after the change in game status therefore it is not updated....

Reply:
Stupid server issues